### DISCUSSION TITLE
Development environment setup in eclipse

### DISCUSSION THREAD

**Discussion by ukrishne** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.generic#124] [Q&A] [answered]
Dear Team,

I am new to java development, i have tried to setup development in eclipse. I just want to make small extension(Eg: "Hello world") like  you have developed many. i just need 30mins of your time via teams call. can you please help me.

Best Regards
Umesha
umesh2007cit@gmail.com

**Comment by grigoriev** [comment, marked as answer]
@ukrishne,
in open-source, it’s all about community and sharing knowledge. 

Instead of a one-on-one call, it’s usually better to post questions in the project’s discussions or issues. That way, others can pitch in, and the answers are available for anyone else who might run into the same issue later.

Feel free to drop your questions, and we’ll help you.
